_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +355,48 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
+$label-color-active: palette(grayscale, x-dark) !default;
+
+$input-background-default: palette(grayscale, white) !default;
+$input-background-default-focus: $input-background-default !default;
+$input-check-radio-background-default: $input-background-default !default;
+$input-check-radio-background-checked: palette(primary, base) !default;
+$input-border-default: palette(grayscale, x-light) !default;
+$input-border-default-focus: palette(primary, base) !default;
+$input-text-default: palette(grayscale-cool, dark) !default;
+$input-text-default-hover: palette(grayscale-cool, x-dark) !default;
+
+$input-background-disabled: palette(grayscale-cool, x-light) !default;
+$input-border-disabled: palette(grayscale-cool, light) !default;
+$input-text-disabled: palette(grayscale-cool, light) !default;
+
+$input-simple-border-focus: palette(grayscale, x-dark) !default;
+$input-simple-background-focus: transparent !default;
+
+$input-slider-track-default: palette(grayscale, x-light) !default;
+$input-slider-border-default: $input-slider-track-default !default;
+$input-slider-border-default-radius: $buttons-border-radius !default;
+$input-slider-handle-default: palette(primary, light) !default;
+$input-slider-handle-default-hover: palette(primary, base) !default;
+$input-slider-default-focus: palette(primary, accent) !default;
+$input-slider-default-hover: palette(primary, x-accent) !default;
+
+$input-slider-track-disabled: palette(grayscale, x-light) !default;
+$input-slider-disabled-border: $input-slider-track-disabled !default;
+$input-slider-handle-disabled: palette(grayscale, base) !default;
+$input-slider-disabled-focus: palette(grayscale, light) !default;
+
+$input-progress-track: palette(grayscale, x-light) !default;
+$input-progress-border: $input-progress-track !default;
+$input-progress-border-radius: $buttons-border-radius !default;
+$input-progress-track-done: palette(primary, accent) !default;
+
+$field-tip-color: palette(grayscale-cool, dark) !default;
```

Comment:
NIT: group the ``$field`` variables together and the ``$input`` variables together, separately.